### ISSUE TITLE
error with MW >= 7.5

### ISSUE THREAD

**Issue by m-haas** [GFZ-Centre-for-Early-Warning/caravan#6]
check accepted magnitude:

Marc:
I had to try several times to get a scenario run through to the server. Other times it got stuck on "Sending input data to server process", but nothing happened.

I tried with a M7.5 at 25km depth (the rest of the settings left unchanged). Just tried again with different depths and it seems related to the magnitude value. Everything >=7.5 leads to this situation. Is it possibly related to a difference between real allowable magnitude range and reported range for which the form validation is applied [5, 7.9]?

**Comment by maxpittore** [comment]
Yes, it definitely depends on a problem in the control of the active area, rather than the magnitude validation. If the reference intensity for M=7.9 is set to Iref=7, then the radius of the active area is less than 300km and the computation starts correctly.
